docs/docs/APIdocs/layers/nn/Loss Functions/AbsCriterion.md

```diff
@@ -0,0 +1,43 @@
+## AbsCriterion ##
+
+**Scala:**
+```scala
+val criterion = AbsCriterion[Float]()
+```
+**Python:**
+```python
+criterion = AbsCriterion()
+```
+
+measures the mean absolute value of the element-wise difference between input
+
+**Scala example:**
+```scala
+import com.intel.analytics.bigdl.nn._
+import com.intel.analytics.bigdl.tensor.Tensor
+val criterion = AbsCriterion[Float]()
+
+val input = Tensor[Float](3).rand()
+val target = Tensor[Float](3).rand()
+val output = criterion.forward(input, target)
+```
+output is
+```
+output: Float = 0.33056465
```

Comment:
This is meanless. Users still can't refer how do you get the output. They don't know the inputs